library/src/main/java/com/github/runly/circularprogressview/CircularProgressDrawable.java

```diff
@@ -199,7 +200,31 @@ private void drawIndeterminate(Canvas canvas) {
 
         canvas.drawArc(mRect, mStartAngle, mSweepAngle, false, mPaint);
       }
-    } else if (mRunState != RUN_STATE_STOPPED) {
+    } else if (isPaused()) {
```

Comment:
Maybe you think one line replaces the other? But it's not the case. The `!isStopped` is checked later on